app/controllers/api/v1/room_settings_controller.rb

```diff
@@ -15,15 +15,17 @@ def show
 
       # PATCH /api/v1/room_settings/:friendly_id
       def update
-        config = MeetingOption.get_config_value(name: room_setting_params[:settingName], provider: current_provider)&.value
-        return render_error status: :bad_request unless config
-
         name = room_setting_params[:settingName]
         value = room_setting_params[:settingValue].to_s
 
+        config = MeetingOption.get_config_value(name:, provider: current_provider)
+        config_value = config[name]
+
+        return render_error status: :bad_request unless config_value
```

Comment:
If we're adding this then I think we may need to add a test for this in the `room_settings_controller_spec`.

Reply:
I'm not sure what you want to test

Reply:
The case where the action will return a `bad_request` response to the client.
Like with the unfound 404 tests.

Reply:
Seems to be unrelated to this PR